aiven/resource_project.go

```diff
@@ -190,14 +197,15 @@ func resourceProjectCreate(_ context.Context, d *schema.ResourceData, m interfac
 			AccountId:        optionalStringPointer(d, "account_id"),
 			BillingCurrency:  billingCurrency,
 			VatID:            vatID,
+			//UseSourceProjectBillingGroup: d.Get("use_source_project_billing_group").(bool),
```

Comment:
Why is this commented out?

Reply:
It shouldn't, and it is my leftover from some tests.